code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -272,6 +272,7 @@
 		),
 		"Grenades" = list(
 			/obj/item/explosive/grenade/frag = 600,
+			/obj/item/explosive/grenade/impact = 300,
```

Comment:
```suggestion
			/obj/item/explosive/grenade/frag = 600,
			/obj/item/explosive/grenade/impact = 50,
```

Reply:
i don't see why not make them alot, theyre similar to HEDP's the only difference is they blow on direct hits on living things when used on GL's, but they do same damage

Reply:
They actually have lower radius than regular grenades when thrown, 3 tiles compared to 4, and single tile light explosion if it hits something along with 2 tiles flash radius.